### DISCUSSION TITLE
Copy branch name from telescope's branches list

### DISCUSSION THREAD

**Discussion by timsofteng** [NeogitOrg/neogit#643] [General]
Hello. Maybe it is implemented already that's why I prefer discussion to issue.
How can I copy branch name under cursor from telescope branches list?

**Comment by CKolkey** [comment]
There's not a way to do that as of yet. Uh... why do you want to?

**Comment by timsofteng** [comment]
@CKolkey in lazygit I did it quite often. Imagine you should push fix to the different environments like dev and prod. It is exactly the same fix. In lazygit I just called first branch to dev like `some-awesome-phenomenal-incredible-filx-dev` copied branch name and then pasted it into popup with new branch for prod. Change one word is a lot easier than retype all branch name.

**Reply by CKolkey** [reply]
Alright, It should be simple to add. I'll see about it later.

**Reply by timsofteng** [reply]
Thanks!
